README.md

```diff
@@ -2,55 +2,54 @@
 
 [![Build Status](https://travis-ci.org/vulcanize/VulcanizeDB.svg?branch=master)](https://travis-ci.org/vulcanize/VulcanizeDB)
 
-### Dependencies
-
+## Dependencies
  - Go 1.9+
- - Postgres 10 
+ - Postgres 10
  - Ethereum Node
    - [Go Ethereum](https://ethereum.github.io/go-ethereum/downloads/) (1.8+)
    - [Parity 1.8.11+](https://github.com/paritytech/parity/releases)
-   
-### Installation 
-`go get github.com/vulcanize/vulcanizedb`
 
-### Setting up the Databases
+## Installation
+`go get github.com/vulcanize/vulcanizedb`
 
+## Setting up the Database
 1. Install Postgres
 1. Create a superuser for yourself and make sure `psql --list` works without prompting for a password.
-1. `createdb vulcanize_private`
+1. `createdb vulcanize_public`
 1. `cd $GOPATH/src/github.com/vulcanize/vulcanizedb`
-1. Import the schema: `psql vulcanize_private < db/schema.sql`
+1. Import the schema: `psql vulcanize_public < db/schema.sql`
 
```

Comment:
If I do the import, then run migrations they fail. Wondering if that impacts the ability to run future migrations. Should we just recommend one way to setup the schema?

Reply:
Good call! I'm leaning toward just recommending setting the schema up by running the migrations. It seems like the [migrate library](https://github.com/mattes/migrate) we're using handles tracking if a migration has been run yet or not.

Reply:
cc93310